presto-main/src/main/java/com/facebook/presto/execution/TaskExecutor.java

```diff
@@ -855,6 +899,16 @@ public long getRunningTasksLevel4()
     }
 
     @Managed
+    public long getLongestSplitTimeNanos()
+    {
+        Iterator<SplitProfile> iterator = splitProfiler.iterator();
+        if (iterator.hasNext()) {
+            return System.nanoTime() - iterator.next().getStartTime();
```

Comment:
Doable but this may fail the unit test given that the granularity gets lost and we may have bunch of tasks running with execution time = 0. Do we wanna remove the assertion in the test or specialize the corner case?